### PR TITLE
[MIG] crm_timesheet: Migration 10.0

### PR DESCRIPTION
**Pull request by chienandalu**
cc @Tecnativa

### CODE REVIEW

File: crm_timesheet/__manifest__.py [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
         'crm',
         'hr_timesheet'
     ],
     'data': [
         'views/crm_lead_view.xml',
-        'views/crm_phonecall_view.xml',
-        'views/hr_analytic_timesheet_view.xml'
+        'views/hr_timesheet_view.xml'
     ],
     'author': 'Antiun Ingeniería S.L.,Odoo Community Association (OCA)',
```
Comment (pedrobaeza) [inline crm_timesheet/__manifest__.py]: Remove Antiun and add Tecnativa

File: crm_timesheet/__manifest__.py [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
     'website': 'http://www.antiun.com',
```
Comment (pedrobaeza) [inline crm_timesheet/__manifest__.py]: Put Tecnativa's website

File: crm_timesheet/models/__init__.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
-# See README.rst file on addon root folder for license details
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
```
Comment (pedrobaeza) [inline crm_timesheet/models/__init__.py]: Remove copyright in `__init__` files

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Review by luismontalba — APPROVED** [review]
Tested on runbot

**Pull request merged**